core/deployment/src/main/java/io/quarkus/deployment/IsDockerWorking.java

```diff
@@ -88,7 +88,7 @@ public Result get() {
                         .loadClass("org.testcontainers.utility.TestcontainersConfiguration");
                 Object configurationInstance = configurationClass.getMethod("getInstance").invoke(null);
                 String oldReusePropertyValue = (String) configurationClass
-                        .getMethod("getUserProperty", String.class, String.class)
+                        .getMethod("getEnvVarOrUserProperty", String.class, String.class)
```

Comment:
There is also a case to be made in using `getEnvVarOrProperty` here, but let's be cautious and only broaden the scope a little instead of all the way